packages/daemon/src/migrations/20200803063219-addDatasetProcess.ts

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
How does the daemon decide the execution order of this kind of migration script

Reply:
Look at:
https://github.com/papb/tory/blob/ed1eb517d403390ba19ba7233bb881f6663cda84/source/helpers/sort-lexicographically.ts#L3

The order is lexicographically.
The require-link is [sequelize/cli](https://github.com/sequelize/cli/blob/master/src/core/migrator.js#L41)->[umzug](https://github.com/sequelize/umzug/blob/master/src/umzug.ts#L333)->[papb/tory](https://github.com/papb/tory/blob/ed1eb517d403390ba19ba7233bb881f6663cda84/source/helpers/sort-lexicographically.ts#L3).